Update evoker_devastation.yaml

awlego opened the pull request:
Updating talent profiles using talent suggestions from wowhead (https://www.wowhead.com/guide/classes/evoker/devastation/talent-builds-pve-dps).

Let me know if there is anything I can do to help verify that I generated the yaml correctly.

awlego committed:
Update evoker_devastation.yaml

Updating talent profiles using talent suggestions from wowhead (https://www.wowhead.com/guide/classes/evoker/devastation/talent-builds-pve-dps).

awlego commented:
I don't actually main devastation, so if these are actually just worse than the profiles that were there feel free to close -- I just saw a comment in your discord and figured I could help out

Bloodmallet commented on bloodytools/simulations/talent_tree_paths/evoker_devastation.yaml:
```diff
@@ -4,9 +4,15 @@
-   - talents=BsbBAAAAAAAAAAAAAAAAAAAAAQDAAAAAAgmGSiIRkkkkDkUSSSLIFkkESSgkkIRIJJ
+M+ AoE w/Unravel and Shattering Star:
+   - talents=BsbBAAAAAAAAAAAAAAAAAAAAAQDCAAAAAAKapIJCJkQSkUSSSLItQikkECSSkIcgkE
+M+ AoE w/Overawe and Cauterizing flame
```
`:` at the end of the line

Bloodmallet commented on bloodytools/simulations/talent_tree_paths/evoker_devastation.yaml:
```diff
@@ -4,9 +4,15 @@
+   - talents=BsbBAAAAAAAAAAAAAAAAAAAAAQDCAAAAAAKaIJikEJkEJlkk0g0CSSkkIkkEJCHIJB
+Raid Pure Single Target Build:
+   - talents=BsbBAAAAAAAAAAAAAAAAAAAAAAkAAAAAAgiWKkIkQCJJJtkENINkIRSIhkEJSkkE
+Raid Raid - General with AoE Burst Build
```
`:` at the end of the line

Bloodmallet commented on bloodytools/simulations/talent_tree_paths/evoker_devastation.yaml:
```diff
@@ -4,9 +4,15 @@
+   - talents=BsbBAAAAAAAAAAAAAAAAAAAAAAkAAAAAAgiWKkIkQCJJJtkENINkIRSIhkEJSkkE
+Raid Raid - General with AoE Burst Build
+   - talents=BsbBAAAAAAAAAAAAAAAAAAAAAAkAAAAAAgiWKSiIJkQSkUSS0g0QiEJhESSkIRSSA
+Raid - Council Cleave Build
```
`:` at the end of the line

Bloodmallet commented on bloodytools/simulations/talent_tree_paths/evoker_devastation.yaml:
```diff
@@ -4,9 +4,15 @@
+   - talents=BsbBAAAAAAAAAAAAAAAAAAAAAAkAAAAAAgiWKSiIJkQSkUSS0g0QiEJhESSkIRSSA
+Raid - Council Cleave Build
+   - talents=BsbBAAAAAAAAAAAAAAAAAAAAAAIAAAAAAoohIJSkkgkDkUSSSDSDJSkECSSSiEJJB
+Raid - Everburning Flame Council Build
```
`:` at the end of the line

Bloodmallet commented on bloodytools/simulations/talent_tree_paths/evoker_devastation.yaml:
```diff
@@ -4,9 +4,15 @@
+Raid - Council Cleave Build
+   - talents=BsbBAAAAAAAAAAAAAAAAAAAAAAIAAAAAAoohIJSkkgkDkUSSSDSDJSkECSSSiEJJB
+Raid - Everburning Flame Council Build
+   - talents=BsbBAAAAAAAAAAAAAAAAAAAAAAIAAAAAAophIJSkIJSkUSSSDSDJSkECSSSiEJJB
```
This build in particular will break when 10.1 is live. It would be great if you update it once more, once the update is almost live.

awlego replied:
Agreed. I'll set a reminder for myself.

awlego replied:
https://github.com/Bloodmallet/bloodytools/pull/63

awlego committed:
Update evoker_devastation.yaml

Added the missing colons at ends of line. Sorry I missed those -- first time working with this repo and I should have seen those.
